src/GRA.Controllers/MissionControl/PerformerManagementController.cs

```diff
@@ -241,6 +243,33 @@ public async Task<IActionResult> PerformerDelete(PerformerListViewModel model)
                 new { page = model.PaginateModel.CurrentPage });
         }
 
+        [HttpPost]
+        public async Task<IActionResult> PerformerSelectionDelete(PerformerSelectionsViewModel model)
+        {
+            var settings = await _performerSchedulingService.GetSettingsAsync();
+            var schedulingStage = _performerSchedulingService.GetSchedulingStage(settings);
+            if (schedulingStage < PsSchedulingStage.SchedulingOpen)
+            {
+                return RedirectToAction(nameof(PerformerManagementController.PerformerSelections), new { id = model.Performer.Id });
+            }
+
+            try
+            {
+                var branchSelection = await _performerSchedulingService.GetBranchProgramSelectionByIdAsync(model.BranchSelectionId);
+                var branch = await _siteService.GetBranchByIdAsync(branchSelection.BranchId);
+                var program = await _performerSchedulingService.GetProgramByIdAsync(branchSelection.ProgramId.Value);
+                branchSelection.IsDeleted = true;
+                await _performerSchedulingService.DeleteBranchSelectionAsync(branchSelection);
+                ShowAlertSuccess($"{branch.Name}'s selection of \"{program.Title}\" is deleted!.");
+            }
+            catch (GraException gex)
+            {
+                ShowAlertDanger("Unable to delete selection: ", gex);
+                return RedirectToAction(nameof(PerformerManagementController.PerformerSelections), new { id = model.Performer.Id });
```

Comment:
Seems like we don't need this line as when it falls out it will do exactly this below.

Reply:
Fixed in 5044ff45f01c4c5f7629dd13bae6f7f7b8c74a0a